apps/wallets/smart-wallet/react/src/app/utils/createAAWallet/createMetamaskWallet.ts

```diff
@@ -24,13 +23,9 @@ export const createMetamaskAAWallet = async (isProd: boolean) => {
                   clientApiKey: process.env.REACT_APP_CROSSMINT_API_KEY_STG || "",
               });
 
-        return await xm.getOrCreateWallet(
-            userIdentifier,
-            isProd ? SmartWalletChain.POLYGON : SmartWalletChain.POLYGON_AMOY,
-            {
-                signer: window.ethereum as any,
-            }
-        );
+        return await xm.getOrCreateWallet(userIdentifier, isProd ? Chain.POLYGON : Chain.POLYGON_AMOY, {
```

Comment:
nit: `xm` => `smartWalletSDK` or like `smartWallets`

Reply:
Yeah demo is a huge mess atm, jon and I will be cleaning up in separate PRs